templates/repo-policy-compliance.service.j2

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Gunicorn instance to serve repo policy compliance endpoints
+After=network.target
+
+[Service]
+User=ubuntu
+Group=www-data
+WorkingDirectory={{working_directory}}
+Environment="GITHUB_TOKEN={{github_token}}"
```

Comment:
For security purposes some or all of these environment variables can be hidden from `systemctl show <service_name>`  using an EnvironmentFile directive pointing to a file with the appropriate permissions.

I understand that this won't be a multiuser system but I saw the word *_TOKEN and I immediately thought of that.

Reply:
I have left some more comments, without using a "review". Sorry for the possible e-mail spamming.

Reply:
I do not think this would increase security. This might not be needed.

Reply:
No Changes